### DISCUSSION TITLE
plans for SCAN

### DISCUSSION THREAD

**Discussion by bentheiii** [dragonflydb/dragonfly#244] [Q&A] [answered]
Hi there, are there any plant to implement the SCAN redis command? I ask because this is the core command used in any redis management software, and if implemented would make DF databases introspectable with redis insight.

EDIT apparently `keys` is supported, my bad

**Comment by romange** [comment, marked as answer]
Hi @bentheiii . Did you try using `scan` and `keys` and it did not work? :)

**Reply by bentheiii** [reply]
I didn't try anything yet but the DF site lists `scan` commands as unsupported

**Reply by romange** [reply]
where do you see it? I look at https://github.com/dragonflydb/dragonfly/blob/main/docs/api_status.md and both marked as supported. 
(`keys` have a safety limit of 8192 that can be overridden by `keys_output_limit` flag).

**Reply by bentheiii** [reply]
I saw in the supported command table in the DF website
https://dragonflydb.io/platform/

**Reply by romange** [reply]
Thanks, we will fix it!
